code/game/objects/structures/plasticflaps.dm

```diff
@@ -1,8 +1,9 @@
-/obj/structure/plasticflaps	//HOW DO YOU CALL THOSE THINGS ANYWAY
+/obj/structure/plasticflaps
 	name = "plastic flaps"
 	desc = "Definitely can't get past those. No way."
-	icon = 'icons/obj/stationobjs.dmi'	//Change this.
+	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "plasticflaps"
+	armor = list(melee = 100, bullet = 80, laser = 80, energy = 100, bomb = 50, bio = 100, rad = 100, fire = 50, acid = 50)
```

Comment:
Well THANKS, phil!
They need to be immune to melee so you can't just hit them with a wrench, aliens can already go through them, and well I guess I'll have to override blob_act() for these, then.